manifests/03-clusterrole.yaml

```diff
@@ -214,6 +214,46 @@ subjects:
   namespace: openshift-insights
   name: gather
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: insights-operator-obfuscation-secret
+  namespace: openshift-insights
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/single-node-developer: "true"
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  resourceNames:
+  - ""
+  verbs:
+  - get
+  - create
+  - update
+  - delete
```

Comment:
Can't we use the existing role https://github.com/openshift/insights-operator/blob/master/manifests/03-clusterrole.yaml#L274 and just extend its rules (to be able to work with secrets)?

Reply:
that is for the operator SA, this is for the gather SA